### ISSUE TITLE
Service account on InstanceGroupManager

### ISSUE THREAD

**Issue by linco95** [pulumi/pulumi-gcp#1790]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

I'm trying to set up our cloud environment by using Pulumi and setting up the proper permissions (deprivileging  all default service accounts, and creating custom roles).
However, our service is using (regional) instance group managers, and in pulumi/gcp I haven't found any option for setting the service account for the manager to use. There seems to be an option to set the service account in the google-native version, but that library is developer preview and I intend to use this setup in production, so that's not an option unfortunately.

The workaround I'm trying out for now is to deprivileging all default accounts, and then explicitly setting the needed permissions/roles on the default accounts.

**tl;dr: Add service account option to `(Region)InstanceGroupManager`.**

I tried searching the issue list here in the repo, but couldn't find anything existing relating to this issue.

**Comment by iwahbe** [comment]
Hi @linco95. Thanks for raising this with us. This issue is tracked upstream as https://github.com/hashicorp/terraform-provider-google/issues/6858.

**Comment by linco95** [comment]
@iwahbe Thank you for directing me to the upstream issue, I didn't consider looking there. It's a problem throughout a lot of the gcp components. I'll keep tracking the issue upstream and continue working with the default account in the meanwhile.
